src/main/java/com/yahoo/sketches/hash/MurmurHash3.java

```diff
@@ -256,12 +256,12 @@ void blockMix128(long k1, long k2) {
      * @return mix
      */
     private static long finalMix64(long h) {
-      h ^= h >>> 33;
-      h *= 0xff51afd7ed558ccdL;
-      h ^= h >>> 33;
-      h *= 0xc4ceb9fe1a85ec53L;
-      h ^= h >>> 33;
-      return h;
+      long result = h ^ h >>> 33;
```

Comment:
It is not necessary to create an additional local variable on the stack!  In fact it would use more stack space and waste a few CPU cycles.  It does not make it any easier to read, so why do this?  Rejected.